### PR TITLE
Makes railings easier to construct, while making them easier to destroy.

### PR DESCRIPTION
**Pull request by iwishforducks**
## About The Pull Request

Changes the cost of railings from 6 metal rods to 2 metal rods.
The time to construct has been reduced from 3.6 to 1 second to be in-line with the grille.
The health of railings has been reduced from 75 to 25.
Armor of railings have been reduced as well by about 30%.

I'm not positive on whether or not it should cost 1 metal rod or 2. I decided to play it safe and make it 2. If some maintainer is interested in making it only cost 1 rod then I will gladly do so- but this was my compromise.

Also changes decaseconds/ticks to seconds in the rod construction code to make it look better.

## Why It's Good For The Game

Railings look nice and it's an absolute pain in the ass that they cost 6 metal rods.

They're also currently substantially stronger than grilles for whatever reason. Grilles have 50 health, while railings have 75.
The armor of railings makes this health of 75 to a whopping effective health of 150.

Railings shouldn't be stronger than full-tile grilles. They should be fairly flimsy.
They also shouldn't take a wrench AND wirecutters to deconstruct. Grilles only take wirecutters and we should mirror that.

## Changelog

:cl:
balance: Railings now only cost 2 rods and are much easier to construct. But they can now be destroyed much easier and cut with wirecutters without unwrenching.
/:cl:

### CODE REVIEW

**Comment by nianjiilical** [comment]
its probably out of scope of this pr but it'd be nice if we could get more expensive reinforced railings, since its nice to use them to cordon off dangerous things

**Comment by Elli-Skala** [comment]
For context they were originally 3 rods but upped to 6 after they got spammed rapidly for the thether + anti grav module  combo which lead to blocked walls

Being able to remove them in one click with wire cutters is a much better solution 👍

**Comment by ophaq** [comment]
This is a good change. Now if only we could construct rail endings too lol.

**Comment by iwishforducks** [comment]
> This is a good change. Now if only we could construct rail endings too lol.

i forgot about this as well i'll fix this too

**Comment by ZephyrTFA** [comment]
change is ok, your gbp makes me sad though

**Comment by iwishforducks** [comment]
> change is ok, your gbp makes me sad though

i have 2 other fix prs up rn if that turns your frown upside down

File: code/game/objects/structures/railings.dm [hunk 1]
```diff
@@ -70,11 +70,10 @@
```
Comment (ArcaneMusic) [inline code/game/objects/structures/railings.dm:70]: Mind adding some screentips to railings while you're here?

**Review by ArcaneMusic — CHANGES_REQUESTED** [review]
Do this and I'll merge by end of today

**Review by ArcaneMusic — APPROVED** [review]

**Pull request merged**